CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+# 4.1.0 (2020-09-11)
+
+- Added the `bindHeightToWindow` input property to the split view component to allow for split views on the main application page to be bound to the window's height. [#37](https://github.com/blackbaud/skyux-split-view/pull/37)
```

Comment:
Does this change meaning?

Added the `bindHeightToWindow` input property to the split view component to bind split views on the main application page to the window's height.